### PR TITLE
Jamf C2C doc (Beta)

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create a new doc for Jamf C2C and the respective release notes.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[JAMF C2C - Documentation](https://sumologic.atlassian.net/browse/CONN-3257)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+1. (Optional) **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [How to Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Value | Required | Description |
+|:--|:--|:--|:--|:--|
+| schemaRef | JSON Object  | `{"type":"Jmaf"}` | Yes | Define the specific schema type. |
```
Comment (shivani-sumo) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md:62]: Correct the typo to Jamf instead jmaf.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+| 24 hours |  [Inventory ID](https://developer.jamf.com/jamf-pro/reference/get_v1-computers-inventory) <ul><li>[Computer History](https://developer.jamf.com/jamf-pro/reference/findcomputerhistorybyid)</li><li>[Computer Management](https://developer.jamf.com/jamf-pro/reference/findcomputermanagementbyid)</li></ul> |
+
+
+## Setup
+
+### Vendor configuration
+
+The Jamf source requires you to provide the Base URL, Client ID, and Client Secret to access the source data.
+
+- The Base URL is used to retrieve the source data from the Jamf. The **Base URL** for Jamf is `https://yourServer.jamfcloud.com`.
```
Comment (shivani-sumo) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md:33]: Could you please add below line after the URL?
`Replace "yourServer" with your organization server.`

**Review by shivani-sumo — APPROVED** [review]

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+| 24 hours |  [Inventory ID](https://developer.jamf.com/jamf-pro/reference/get_v1-computers-inventory) <ul><li>[Computer History](https://developer.jamf.com/jamf-pro/reference/findcomputerhistorybyid)</li><li>[Computer Management](https://developer.jamf.com/jamf-pro/reference/findcomputermanagementbyid)</li></ul> |
+
+
+## Setup
+
+### Vendor configuration
+
+The Jamf source requires you to provide the Base URL, Client ID, and Client Secret to access the source data.
+
+- The Base URL is used to retrieve the source data from the Jamf. The **Base URL** for Jamf is `https://yourServer.jamfcloud.com`, replace `yourServer` with your organization server.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md:33]:
```suggestion
- The Base URL is used to retrieve the source data from Jamf. The **Base URL** for Jamf is `https://yourServer.jamfcloud.com`. Replace `yourServer` with your organization server.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+1. In **Base URL**, enter your Jamf instance domain, `https://yourServer.jamfcloud.com`.
+1. In **Client ID**, enter the Client ID you generated from the Jamf platform.
+1. In **Client Secret**, enter the Client Secret you generated from the Jamf platform.
+1. (Optional) The **Polling Interval** is set for 24 hours by default. You can adjust it based on your needs.
+1. (Optional) **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [How to Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jamf-source.md:58]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

**Review by jpipkin1 — APPROVED** [review]
Looks good @JV0812! I have only minor comments. And I added the new article to the Product List article (see commit "[Add Jamf Source to product list](https://github.com/SumoLogic/sumologic-documentation/pull/4165/commits/a9462215fdd184d977abe6904119bb0b28418810)").

File: blog-service/2024-06-07-collection.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Jamf (Collection) 
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - cloud-to-cloud
+  - integrations
+  - collection
```
Comment (kimsauce) [inline blog-service/2024-06-07-collection.md:7]:
```suggestion
  - collection
  - jamf
```

File: blog-service/2024-06-07-collection.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: Jamf (Collection)
```
Comment (kimsauce) [inline blog-service/2024-06-07-collection.md:2]:
```suggestion
title: Jamf C2C Source (Collection) 
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Nice work! Had a couple minor edits to help boost SEO.

**Comment by JV0812** [comment]
@jpipkin1 Sorry, I missed it. Thanks for helping me with that and for your review.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**